libs/utils/utils.js

```diff
@@ -419,6 +419,40 @@ export function decorateSVG(a) {
     return a;
   }
 }
+function hasUrlWithPipe(str) {
+  const regex = /^https?:\/\/.*\|.*/;
+  return regex.test(str);
+}
+
+export function decorateImageLinks(a) {
+  const images = a.querySelectorAll('img');
+  if (!images.length) return a;
+  [...images].forEach((img) => {
+    // Check if the image has an alt tag with a url & pipe - 'https... |'
+    if (!(hasUrlWithPipe(img.alt))) return a;
+    try {
+      // //Mine for URL and alt text
+      const splitText = img.alt.split('|');
+      const textUrl = new URL(splitText.shift().trim());
+      const altText = splitText.join('|').trim();
+      if (altText) img.alt = altText;
+
+      const pic = img.closest('picture');
+      const picParent = pic.parentElement;
+      const aTag = createTag('a', { href: textUrl, class: 'desc-link' }, pic);
+      if (aTag.href.includes('#_blank')) {
+        aTag.setAttribute('target', '_blank');
+        aTag.href = aTag.href.replace('#_blank', '');
+      }
+      picParent.append(aTag);
```

Comment:
```suggestion
      pic.parentElement?.append(aTag);
```
I don't think there's a need to cache the parent element, it's only used once

Reply:
In testing it turns out this was necessary to declare the picParent as a valid DOM element.
I tried this suggestion and it resulted in a runtime error.